examples/extensions-zalando/README.md

```diff
@@ -13,6 +13,13 @@ Developing the Swagger Plugin is easy, just execute the following command:
 
 This will start IntelliJ IDEA with the plugin installed.
 
+###
+Check dependency updates (preferably replaced with Dependabot:
```

Comment:
Any reason why you don't just directly go with dependabot? Is it to avoid getting 20+ PR's for each individual dependency, so better to bring it up to speed and then maybe turn it on?

Reply:
Haven't yet checked what needs to be done to get Dependabot set up on github.com-hosted project (instead of Zalando internal GH). So before looking at that thought to just get the dependencies updated to start with.

Reply:
Ah okay. But I guess, it is still good practice to get the dependencies to the current state before enabling it as otherwise, as I said, you get a lot of PR's.